set.mm

```diff
@@ -528868,6 +528934,405 @@ Set induction (or epsilon induction)
+      UYTSZGUKZUWPKZCLVUEVVAUXKVVFCVVAUXJVVEUWPVVAUXJUVRUYGUJZUWDUYFUVROZUVIPZH
+      ZUYOUVROZUYQUVROZAQZUEUYFRZSZDUKVVEVVAUXIVVODVVAUXEVVGUXGVVJUXHVVNVVAUXDU
+      YGUVRVVAUXCUYFPUXDUYGPUXBUYETZUXCUYFTURUOVVAUXFVVIUWDVVAUXCUYFUVIUVRVVPVJ
+      USVVAUXHUWKFUYFRVVNVVAUWKFUXCUYFVVPVCUWKVVMFUEUYFFUEVPZUWHVVKUWJVVLAUWGUY
+      OUVRVEVVQUWIUYQUVRUWGUYOTVFVGZVQVIVKVLVVOVVDDGDGVPZVVGUYHVVJVVCVVNUYTUYGU
+      VRUYDVRVVSUWDUYJVVIVVBVVSUWCUYIUVHNUVRUYDVTWAVVSVVHUYKUVIUYFUVRUYDVTWAVSV
+      VSVVMUYSUEUYFVVSVVKUYPVVLUYRAUYOUVRUYDVTUYQUVRUYDVTVGWBVKWCVIVMVNVVFVUDCU
+      DCUDVPZVVEVUBUWPVUCVVTVVDVUAGVVTVVCUYNUYHUYTVVTVVBUYMUYJUVIUYLUYKWDUSWEVL
+      UVIUYLUVHBWJWFWGVIVOUXBUYFPZUXLVUOUVGVWAUXKVUNCVWAUXJVUMUWPVWAUXIVULDVWAU
+      XEVUGUXGVUJUXHVUKVWAUXDVUFUVRVWAUXCUYGPUXDVUFPUXBUYFTZUXCUYGTURUOVWAUXFVU
+      IUWDVWAUXCUYGUVIUVRVWBVJUSVWAUWKFUXCUYGVWBVCVKVLVMVNVOUBEVPZUXLUWTUVGVWCU
+      XKUWSCVWCUXJUWNUWPVWCUXIUWMDVWCUXEUWBUXGUWFUXHUWLVWCUXDUWAUVRVWCUXCUVTPUX
+      DUWAPUXBUVSTZUXCUVTTURUOVWCUXFUWEUWDVWCUXCUVTUVIUVRVWDVJUSVWCUWKFUXCUVTVW
+      DVCVKVLVMVNVOUVGUYBCUVGUXTUWPDUXTUVHUVIAQZUVGUWPUXRUXSVWEUXOUXRUXSVWEUWCU
+      VHUXPUVIAWHWKWIUVDVWEUWPKUVFABUVHUVIWLWMWNWOWPUYEULJZUVGVUEVUOVWFUVGVUEVU
+      OVWFUVGVUESZVUNCVWGVULUWPDVWFUVGVUEVULUWPKZVUEUYHUYJUYKVVHPZHZUYTSZGUKZUV
+      HVVHBQZKZVWFUVGHZVWHVUDVWNUDVVHUYFUVRWQZUYLVVHPZVUBVWLVUCVWMVWQVUAVWKGVWQ
+      UYNVWJUYHUYTVWQUYMVWIUYJUYLVVHUYKWDUSWEVLUYLVVHUVHBWJZWFXAVWOVULVWNUWPVWO
+      VULVWNUWPKVWOVULHZVWLVWMUWPVWSUVRUYGXBZUYGUJZNVWTOZUVHPZUYFVWTOZVVHPZUYOV
+      WTOZUYQVWTOZAQZUEUYFRZVWLVWSVUGUYGVUFUFVXAVWOVUGVUJVUKWRUYGWSVUFUYGUVRWTX
+      CVWSVXBUWCUVHVWSNUYGUVRVWSUYFXLZNUYGJVWSUYFULJZVXJVWFVXKUVGVULUYEXDXEUYFX
+      FURZUYFXGURXHUWDVUIVUGVUKVWOXIXJUYFUYGJZVXEVWSUYFUYEUCXMXKXNZUYFUYGUVRYCX
+      OVWSVXHUEUYFVWSUYOUYFJZHZVVKVVLVXFVXGAVXPUWKVVMFUYGUYOVVRVUGVUJVUKVWOVXOX
+      PVXOUYOUYGJVWSUYOUYFXQXRZXSVXPUYOUYGUVRVXQXHVXPUYQUYGUVRVWSVXOUYQUYGJZVWS
+      VXJVXOVXRXTVXLUYOUYFYAURWKXHYBYDVWKVXAVXCVXEHZVXISGVWTUVRUYGDXMYEUYDVWTPZ
+      UYHVXAVWJVXSUYTVXIUYGUYDVWTVRVXTUYJVXCVWIVXEVXTUYIVXBUVHNUYDVWTVTWAVXTUYK
+      VXDVVHUYFUYDVWTVTWAVSVXTUYSVXHUEUYFVXTUYPVXFUYRVXGAUYOUYDVWTVTUYQUYDVWTVT
+      VGWBVKYFYGVWSVWMVVHUVIBQZUWPVWSVVHVUHUVIBVXMVWSUWHUWJBQZFUYGRZVVHVUHBQZVX
+      NVWSUVDVUKVYCVWFUVDUVFVULYHVWOVUGVUJVUKYIUVDUWKVYBFUYGABUWHUWJWLYJYKVYBVY
+      DFUYFUYGUWGUYFPZUWHVVHUWJVUHBUWGUYFUVRVEVYEUWIUYGUVRUWGUYFTVFVGYMYNUWDVUI
+      VUGVUKVWOYLYOVWMVYAHZUVHUVIUVEQZVWSUWPVYFVUCUYLUVIBQZHZUDUKVYGVYIVYFUDVVH
+      VWPVWQVUCVWMVYHVYAVWRUYLVVHUVIBYPVSYFUDUVHUVIBBUAXMCXMUUDYQVWSUVEBUVHUVIV
+      WFUVDUVFVULUUAUUEWNUUBUUCUUFUUGWNUUHWOWPUUPUUIUUJUUKUULUXAUWSEULRZCLUWRUW
+      SECULUUMVYJUWQCUWNUWPEULUUNYRUUOYSUWQUVNCUWOUVLUWPUVMUWOUVHUVIUVKQUVLUVHU
+      VIADEFUUQUVHUVIUVKYTUURUVHUVIBYTUUSYRYSWPUVKUUTUVQUVPXTAUVAUACUVKBUVBUVCY
+      Q $.
+  $}
+
+  ${
+    $d R x y f n a $.
+    $( Calculate the domain of a transitive closure.  (Contributed by Scott
```

Comment:
Maybe "The domain of a transitive closure of a clas is the domain of the class."

Reply:
Done